### PR TITLE
touchups

### PR DESCRIPTION
**Pull request by mejango**

### CODE REVIEW

File: contracts/JBProjects.sol [hunk 1]
```diff
@@ -38,27 +37,21 @@ contract JBProjects is ERC721Votes, Ownable, IJBProjects, JBOperatable {
...
+    _projectId The ID of the project to which the metadata belongs.
     _domain The domain within which the metadata applies.
   */
   mapping(uint256 => mapping(uint256 => string)) public override metadataContentOf;
 
+  /**
+    @notice
+    The contract resolving each project ID to its ERC721 URI.
+  */
+  IJBTokenUriResolver public override tokenUriResolver;
```
Comment (odd-amphora) [inline contracts/JBProjects.sol:53]: For consistency should we s/Uri/URI everywhere (contract name, local variable, etc.)?

File: contracts/JBProjects.sol [hunk 2]
```diff
@@ -140,18 +133,17 @@ contract JBProjects is ERC721Votes, Ownable, IJBProjects, JBOperatable {
   */
   function tokenURI(uint256 _projectId) public view override returns (string memory) {
```
Comment (odd-amphora) [inline contracts/JBProjects.sol:134]: ^ here we use all caps (my preference)

File: contracts/JBProjects.sol [hunk 3]
```diff
@@ -140,18 +133,17 @@ contract JBProjects is ERC721Votes, Ownable, IJBProjects, JBOperatable {
...
     @notice 
-    Set the address of the IJBTokenUriResolver used to retrieve the tokenURI of projects
+    Set the address of the IJBTokenUriResolver used to retrieve the tokenURI of projects.
 
     @param _newResolver The address of the new resolver.
   */
   function setTokenUriResolver(IJBTokenUriResolver _newResolver) external override onlyOwner {
     tokenUriResolver = _newResolver;
-    emit SetTokenUriResolver(_newResolver);
+    emit SetTokenUriResolver(_newResolver, msg.sender);
```
Comment (odd-amphora) [inline contracts/JBProjects.sol:147]: Same with event and this fn name.

File: contracts/JBProjects.sol (hunk 2 above)
Comment (mejango) [reply contracts/JBProjects.sol:134]: this is overriden and part of the base contract. everywhere else we use Uri

File: contracts/JBProjects.sol (hunk 3 above)
Comment (mejango) [reply contracts/JBProjects.sol:147]: this is the standard. we cant change the overriden tokenURI one though. The rest of the JB ecosystem prefers the camel case version.

should we keep consistent in the contract, or in the JB ecosystem?

Comment (odd-amphora) [reply contracts/JBProjects.sol:147]: Ah yes you're right, sorry forgot about that. camelCaseSoundsGoodToMe

**Review by odd-amphora — APPROVED** [review]

**Pull request merged**